public/js/reducers/atomListReducer.js

```diff
@@ -0,0 +1,12 @@
+
+
+export default function atomList(state = null, action) {
+    switch (action.type) {
+
+        case 'ATOM_LIST_GET_RECEIVE':
+            return action.atomList || false;
```

Comment:
Would it make sense to default to `[]` at the start and on recieve rather than false/null, the propTypes check is looking for an array only